cpp/IceGrid/icebox/greeter-hall.xml

```diff
@@ -2,15 +2,17 @@
   <!-- An application describes nodes and servers deployed on these nodes. An IceGrid deployment (a registry plus one or
   more nodes) can host several applications. -->
   <application name="GreeterHall">
+    <!-- Define a variable named 'icebox', which refers to the icebox executable. -->
+    <variable name="icebox" value="icebox"/>
```

Comment:
We should mention how this works, we later override this on the command line for Windows